packages/site/package.json

```diff
@@ -50,13 +48,20 @@
     "@tailwindcss/typography": "^0.5.9",
     "@types/react": "^18.0.29",
     "@types/react-dom": "^18.0.11",
+    "autoprefixer": "^10.4.16",
+    "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^6.7.3",
+    "html-webpack-plugin": "^5.5.3",
+    "postcss": "^8.4.31",
+    "postcss-loader": "^7.3.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-markdown": "^8.0.6",
     "react-router-dom": "^6.10.0",
     "style-loader": "^3.3.2",
-    "tailwindcss": "^3.3.0"
+    "tailwindcss": "^3.3.3",
+    "typescript": "^5.0.2",
```

Comment:
monorepo has typescript as a devDependency in root. do we still need it here?

Reply:
Good call, you're totally right - removed the `typescript` dependency from there